packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -173,30 +173,30 @@ abstract class HttpRequestData extends NetworkRequest {
   }
 
   /// All instant events logged to the timeline for this HTTP request.
-  List<HttpInstantEvent> get instantEvents;
+  List<HttpInstantEvent>? get instantEvents;
 
   /// A list of all cookies contained within the request headers.
   List<Cookie> get requestCookies;
```

Comment:
should this be nullable too?

Reply:
Nope. Actually both can be non nullable because they can be empty lists.